eservice-versions-archiver/src/main/resources/application.conf

```
@@ -0,0 +1,23 @@
+eservice-versions-archiver {
+    queue {
+        archiving-eservices-queue-url   = ${ARCHIVING_ESERVICES_QUEUE_URL}
+        max-number-of-messages-per-file = 1000
+        max-number-of-messages-per-file = ${?MAX_NUMBER_OF_MESSAGES_PER_FILE}
+        visibility-timeout-in-seconds   = 60
+        visibility-timeout-in-seconds   = ${?QUEUE_VISIBILITY_TIMEOUT_IN_SECONDS}
+    }
+
+    services {
+        catalog-process = ${CATALOG_PROCESS_URL}
+    }
+
+    read-model {
+        connection-string   = "mongodb://"${READ_MODEL_DB_USER}":"${READ_MODEL_DB_PASSWORD}"@"${READ_MODEL_DB_HOST}":"${READ_MODEL_DB_PORT}"/"${READ_MODEL_DB_NAME}"?directConnection=true&serverSelectionTimeoutMS=2000"
```

these configuration parameters work only from local environment.
The correct params are
```suggestion
        connection-string   = "mongodb://"${READ_MODEL_DB_USER}":"${READ_MODEL_DB_PASSWORD}"@"${READ_MODEL_DB_HOST}":"${READ_MODEL_DB_PORT}"/"${READ_MODEL_DB_NAME}"?replicaSet=rs0&readPreference=secondaryPreferred"
```

---

I see, then I need to fix it in the [tenantsAttributeChecker's job](https://github.com/pagopa/interop-be-jobs/blob/0f10a1070c113abc43186f16fe15e1a7514bbe1c/tenants-attributes-checker/src/main/resources/application.conf#L13) too